roles/ipareplica/library/ipareplica_install_ca_certs.py

```diff
@@ -297,39 +297,40 @@ def main():
     config.ca_host_name = config_ca_host_name
     config.ips = config_ips
 
-    remote_api = gen_remote_api(config.master_host_name, paths.ETC_IPA)
-    installer._remote_api = remote_api
+    if install_ca_cert is not None:
```

Comment:
Wouldn't it be better to do something like this instead?

```
if install_ca_cert is None:
    ansible_module.exit_json(changed=False,
                             config_master_host_name=config.master_host_name,
                             config_ca_host_name=config.ca_host_name)
```

It would be a small change and **changed** would be False.

Reply:
fixed.